examples/response/ldp_vp.json

```diff
@@ -46,7 +46,7 @@
     "type": "Ed25519Signature2018",
```

Comment:
This example is old, and should be updated to support https://www.w3.org/TR/vc-data-integrity/#dataintegrityproof

Reply:
that's kinda outside the scope of this PR though

Reply:
#487

Reply:
Could you please fix it in this PR, tho, Brian....? given the time pressure, etc...? don't think it is that big of a scope creep...

Reply:
I also don't know how to fix it or if it actually needs fixing. Being old doesn't make it wrong.

Reply:
I don't have the knowledge of any of the W3C VC WG/CG stuff to know if or what fixing is needed or appropriate and I'm definitely not going to acquire that knowledge anytime soon.

Reply:
Doing this here in this PR will not help anyone or speed up anything.

Reply:
one less issue and one less PR in my view. but sure, I will bug you all to do and/or review that editorial PR when the time comes